cg/meta/upload/scout/hk_tags.py

```diff
@@ -21,6 +21,7 @@ class CaseTags(BaseModel):
     peddy_check: Set[str] = Field(None, description="Peddy pedigree check, only for rare disease")
     multiqc_report: Set[str]
     delivery_report: Set[str]
+    coverage_qc_report: Set[str] = Field(None, description="Static coverage report for cancer")
```

Comment:
This is same/similar to delivery_report for rare disease so I suggest that we reuse that field here